config/casestudies/hdi.yml

```diff
@@ -198,7 +200,9 @@ asyncapi: #More specific details about AsyncAPI itself and how is it used.
   usecase: |
     - The AsyncAPI documents are used for documentation purposes by the platform team. 
       It provides a comprehensive overview of the asynchronous communication in our system, including the available topics and the structure of the messages.
-    - (Future Plans) The AsyncAPI document will be used to generate Java DTOs using the AsyncAPI Java generator, ensuring type safety and clear data structure understanding.
+    - (Outlook) The AsyncAPI document will be used to generate Java DTOs using the AsyncAPI Java generator, ensuring type safety and clear data structure understanding.
```

Comment:
for simple java class generation, instead of generator, I suggest modelina https://github.com/asyncapi/modelina

AsyncAPI Generator and templates developers rather think of clients, docs, sdk, and stuff like that - for classes, we found out some time ago that better approach is to separate it into separate library.

so best maybe just write that outlook is to generate DTOs, and not much important what will you use for it

Reply:
```suggestion
    - (Outlook) The AsyncAPI document will be used to generate Java DTOs, ensuring type safety and clear data structure understanding.
```